docs/components/radio/_spec.scss

```diff
@@ -0,0 +1,65 @@
+@import "components/radio/config";
+
+.radio-spec {
+  hx-radio-control {
+    > label {
+      > hx-radio {
+        display: none;
+      }
+    }
+
+    > input[type="radio"] {
+      &.mock-focus {
+        @include hxRadioControl($pseudo-state: focus);
+
+        ~ label > hx-radio {
+          @include hxRadioFacade($pseudo-state: focus);
+        }
+      }
+
+      &.mock-hover {
+        @include hxRadioControl($pseudo-state: hover);
+
+        ~ label > hx-radio {
+          @include hxRadioFacade($pseudo-state: hover);
+        }
+      }
+
+      &:disabled.mock-focus {
+        @include hxRadioControl(disabled, focus);
+      }
+
+      &:disabled.mock-hover {
+        @include hxRadioControl(disabled, hover);
+
+        ~ label > hx-radio {
+          @include hxRadioFacade(disabled, hover);
+        }
+      }
+    }
+  }
+
+  // INVALID STATE
+  hx-radio-set[hx-dirty] input[type="radio"]:enabled:invalid,
+  hx-radio-set.hxInvalid input[type="radio"]:enabled {
+    &.mock-focus {
+      @include hxRadioControl(invalid, focus);
+    }
```

Comment:
### Please notice the cyan focus ring bleeding through on the invalid styles:

<img width="793" alt="Screen Shot 2020-01-28 at 1 01 30 PM" src="https://user-images.githubusercontent.com/10120600/73296422-534fdd00-41cf-11ea-9fdc-72a9ab1337ca.png">

* This is a result of the invalid styles not being applied to `<hx-radio>`.   Please add this inside the `&.mock-focus` selector:
```scss
  ~ label > hx-radio {
    @include hxRadioFacade(invalid, focus);
  }
```

Reply:
Changes have been done, please review.